sssd_test_framework/roles/__init__.py

```diff
@@ -1,2 +1,3 @@
 """SSSD multihost roles."""
+
```

Comment:
Looks like your IDE is updating this file?

Reply:
Yes, this was the same as the hosts/init.py. Let's get an answer there before I drop this one.

Reply:
dropped the commit